frontend/src/components/SkillGroup/SkillsGroup.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+
+import SkillItem from './SkillItem';
+import { ISkillsGroupProps } from './types';
+import { SkillsGroupWrapper, SkillsTitle, SkillsBox, SkillsDivider } from './styled';
+
+const SkillsGroup: FC<ISkillsGroupProps> = ({ skillFounded, skills }) => {
```

Comment:
the folder's name is 'SkillGroup', so file and component should also have name 'SkillGroup', not 'SkillsGroup'